src/k8s-extension/azext_k8s_extension/vendored_sdks/_version.py

```diff
@@ -1,9 +1,8 @@
 # coding=utf-8
 # --------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for license information.
-# Code generated by Microsoft (R) AutoRest Code Generator.
-# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
 # --------------------------------------------------------------------------
 
-VERSION = "2.0.0"
+VERSION = "0.1.0"
```

Comment:
May I ask if it is better to use `2.1.0` instead of `0.1.0` as the version?

Reply:
this file was replaced as a part of the generated files from the swagger sdk.